services/catarse/app/models/contribution.rb

```diff
@@ -203,7 +207,9 @@ def contribution_attributes
         minimum_value: reward.minimum_value
       } : nil,
       contribution_email: user.email,
-      slip_url: payment && payment.slip_payment? ? payment.gateway_data['boleto_url'] : nil
+      slip_url: payment && payment.slip_payment? ? payment.gateway_data['boleto_url'] : nil,
+      pix_qr_code: payment && payment.pix_payment? ? QRCode::Renderer.as_svg(payment.gateway_data['pix_qr_code']) : nil,
+      pix_code: payment && payment.pix_payment? ? payment.gateway_data['pix_qr_code'] : nil
```

Comment:
**Dúvida**
Aqui ficou um pouco confuso, o `pix_qr_code` do `gateway_data` não representa o qr_code, na verdade seria a chave do pix, né isso? Se for, não seria melhor chamar de `pix_key` ou coisa assim? Pois na linha de cima tem:
`pix_qr_code: Render da chave como qr code`, e na linha de baixo:
`pix_code: pix_qr_code`.